community/examples/slurm-gcp-v5-ubuntu2004.yaml

```diff
@@ -0,0 +1,85 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+blueprint_name: slurm-gcp-v5-ubuntu2004
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: slurm-gcp-v5
+  region: us-central1
+  zone: us-central1-c
+  slurm_cluster_name: ghpc
```

Comment:
The PR that removes this as requirement has been merged, right? This could be removed from the example as well.

Reply:
good catch!